tests/python/test_mset_concurrent.py

```
@@ -0,0 +1,362 @@
+        # 并发执行删除和设置
+        with ThreadPoolExecutor(max_workers=2) as executor:
+            future1 = executor.submit(delete_operation)
+            future2 = executor.submit(mset_operation)
+            future1.result()
+            future2.result()
+        
+        # 操作应该都成功完成（不崩溃）
+        # 最终状态可能是存在或不存在，但不应该有部分状态
+        values = r.mget(test_keys)
+        
+        # 要么全部存在，要么全部不存在（原子性）
```

Ruff (0.14.4)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)